ozark/pom.xml

```diff
@@ -79,13 +120,37 @@
                     </execution>
                 </executions>
             </plugin>
+            <!--
+             This plugin is reponsible for packaging artifacts
+             as OSGi bundles.  Please refer to
+             http://felix.apache.org/site/apache-felix-maven-bundle-plugin-bnd.html
+             for more information about how to use this plugin.
+           -->
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
```

Comment:
The warning is quite clear:
```
[WARNING]
[WARNING] Some problems were encountered while building the effective model for org.glassfish.ozark:ozark:bundle:1.0.0-m03-SNAPSHOT
[WARNING] 'build.plugins.plugin.version' for org.apache.felix:maven-bundle-plugin is missing. @ org.glassfish.ozark:ozark:[unknown-version], .../pom.xml, line 129, column 21
[WARNING] 'build.plugins.plugin.version' for org.glassfish.build:spec-version-maven-plugin is missing. @ org.glassfish.ozark:ozark:[unknown-version], .../pom.xml, line 33, column 21
[WARNING]
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING]
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING]
```